src/dashboard/Push/PushNew.react.js

```diff
@@ -201,27 +213,39 @@ export default class PushNew extends DashboardView {
     if (!!changes.increment_badge) {
       payload.badge = "Increment";
     }
-    Parse.Push.send({
-      where: changes.target || new Parse.Query(Parse.Installation),
-      data: payload,
-    }, {
-      useMasterKey: true,
-    }).then(({ error }) => {
-      //navigate to push index page and clear cache once push store is created
-      if (error) {
-        promise.reject({ error });
-      } else {
-        //TODO: global success message banner for passing successful creation - store should also be cleared
-        const PARSE_SERVER_SUPPORTS_PUSH_INDEX = false;
-        if (PARSE_SERVER_SUPPORTS_PUSH_INDEX) {
-          history.push(this.context.generatePath('push/activity'));
+    
+    if(changes.push_time_type === 'now') {
```

Comment:
Not to be nit-picky, and obviously not high priority, can you add a space between `if` and `(` everywhere that you've omitted it? For consistency.